docs/en/observability/apm-ui/troubleshooting.asciidoc

```diff
@@ -42,6 +42,8 @@ This section can help with any of the following:
 * You see a message like "No Services Found",
 * You see errors like "Fielddata is disabled on text fields by default..."
 
+// Assuming the APM integration is no longer responsible for index templates and ingest pipelines,
```

Comment:
good question. I don't think this "common problem" due to missing integration package applies to 8.15+ apm.